k8s/scripts/cis-yaml-to-md.py

```diff
@@ -0,0 +1,115 @@
+import argparse
+import logging
+from pathlib import Path
+
+import yaml
+from jinja2 import Environment, FileSystemLoader
+
+USAGE = """
+python3 cis-yaml-to-md.py --input-directory=INPUT_DIR --output-directory=OUTPUT_DIR
+"""
+
+DESCRIPTION = """
+Parse the YAML files in the input directory and generate
+Markdown files in the output directory. The Markdown files are generated using the
+Jinja2 template file.
+
+It is expected that the input directory contains a config.yaml file. This file
```

Comment:
it might be handy to suggest where this comes from (I assume it is the cis config.yaml from https://raw.githubusercontent.com/canonical/kube-bench/ck8s/cfg/config.yaml)